app/helpers/ui_helper.rb

```diff
@@ -0,0 +1,43 @@
+module UiHelper
```

Comment:
Good point - I thought helpers were more general, hence the name, but let's do that.